vitest.config.js

```diff
@@ -3,19 +3,24 @@ import { defineConfig, configDefaults } from 'vitest/config'
 export default defineConfig({
   test: {
     globals: true,
-    environment: 'node',
+    include: ['**/test/**/*.test.js'],
     clearMocks: true,
     coverage: {
       provider: 'v8',
-      reportsDirectory: './coverage',
-      reporter: ['text', 'lcov'],
-      include: ['src/**'],
+      reportOnFailure: true,
+      clean: false,
+      reporter: ['lcov'],
+      include: ['src/**/*.js'],
       exclude: [
         ...configDefaults.exclude,
         '.public',
         'coverage',
         'postcss.config.js',
-        'stylelint.config.js'
+        'stylelint.config.js',
+        '**/node_modules/**',
```

Comment:
`node_modules` are included in the `configDefaults` so no need to double exclude them